exercise/tests/test_helpers.py

```diff
@@ -36,3 +38,55 @@ def test_copy_exercise(self):
         )
 
         self.assertEqual(1, Exercise.objects.filter(copied_from=exercise).count())
+
+    def test_trial_exercise(self):
+        """
+        Testing helper method that make a trial of an exercise
+        """
+        self.organization = OrganizationFactory()
+        self.user = UserFactory(
+            username="user_with_organization", organization=self.organization
+        )
+        exercise = ExerciseFactory(organisation=self.organization)
+        trial_exercise = add_trial(exercise, self.user)
+
+        self.assertIsNotNone(trial_exercise)
+        self.assertEqual(2, Exercise.objects.all().count())
+
+        self.assertEqual(str(exercise.id), str(trial_exercise.copied_from.id))
+        self.assertEqual(2, trial_exercise.trial_version)
+
+        self.assertEqual(
+            exercise.demographics.all().count(),
+            trial_exercise.demographics.all().count(),
+        )
+        self.assertEqual(
+            exercise.emails.all().count(), trial_exercise.emails.all().count()
+        )
+        self.assertEqual(
+            exercise.files.all().count(), trial_exercise.files.all().count()
+        )
+
+        self.assertEqual(1, Exercise.objects.filter(copied_from=exercise).count())
+
+    def test_trial_exercises(self):
+        """
+        Testing helper method that make trials of an exercise
+        """
+        self.organization = OrganizationFactory()
+        self.user = UserFactory(
+            username="user_with_organization", organization=self.organization
+        )
+        exercise = ExerciseFactory(organisation=self.organization)
+        trial_exercise = add_trial(exercise, self.user)
+        another_trial_exercise = add_trial(exercise, self.user)
+
+        self.assertIsNotNone(trial_exercise)
+        self.assertIsNotNone(another_trial_exercise)
+        self.assertEqual(3, Exercise.objects.all().count())
+
+        self.assertEqual(str(exercise.id), str(trial_exercise.copied_from.id))
+        self.assertEqual(str(exercise.id), str(another_trial_exercise.copied_from.id))
+
+        self.assertEqual(2, Exercise.objects.filter(copied_from=exercise).count())
+        self.assertEqual(2, Exercise.objects.filter(initial_trial=exercise).count())
```

Comment:
same here check that both new trials `initial_trial` is set to `exercise.id` pls